experimentalServer/experimentalServer.go

```diff
@@ -25,7 +28,16 @@ type ResponseJSON struct {
 	FDNS_A []string
 	RDNS   []string
 }
-
+// load config
+func GetMeta(path string) (MetaConfig *MetaJSON) {
+	MetaConfig = new(MetaJSON)
+	data, err := ioutil.ReadFile(path)
+	if err != nil {
+		log.Fatalf("Error opening config file: %v", err)
+	}
+	json.Unmarshal(data, &MetaConfig)
```

Comment:
If you don't mind, can you check the return on `json.Unmarshal` and return a fatal error if it fails?
https://golang.org/pkg/encoding/json/#Unmarshal

Reply:
Yep! I thought I had done that 😂